src/artstationapplication/ArtStationApplication.java

```diff
@@ -24,958 +25,992 @@
 import javafx.scene.layout.FlowPane;
 import javafx.scene.layout.HBox;
 
-
 //import java.awt.Toolkit;
 
-
 /**
  *
  * @author wilder4690
  */
```

Comment:
The deleting everything and replacing it all is problematic, because it makes it difficult to see what exactly was changed. Is this something that is going to happen every time? Is there anything I can do on my end to avoid this? Or is it because I was using netBeans IDE and you were using Eclipse?

Reply:
Yes, it was using eclipse's code formatting, I'll fix another pull without the full overwrite soon